portal-kernel/src/com/liferay/portal/kernel/listtype/manager/ListTypeEntryManagerUtil.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.kernel.listtype.manager;
+
+import com.liferay.portal.kernel.module.service.Snapshot;
+
+/**
+ * @author Mátyás Wollner
+ */
+public class ListTypeEntryManagerUtil {
+
+	public static String fetchListTypeEntry(
+		long listTypeDefinitionId, String key) {
+
+		ListTypeEntryManager listTypeEntryManager =
+			_listTypeEntryManagerSnapshot.get();
+
```

Comment:
please check that listTypeEntryManager is not null (like in `PortalDefaultPermissionsConfigurationManagerUtil`)